data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg

```diff
@@ -862,6 +888,10 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
     movement_type=quenoth_foot
     movement=6
     hitpoints=32
+    # The evolution of this unit is carried out only with AMLAs so that it takes two amlas
+    # to reach level two then two more for level 3.
+    # As the unit variables do not include the number of amla achieved,
+    # the maximum experience is the only one that can be directly used in advancement without resorting to an external lua type function.
```

Comment:
Please also add:
`# Do not modify this value without also modifying the max_experience checks in the custom AMLAs`

Reply:
it's done